tutorials/fiori-ios-scpms-starter-mission-06/fiori-ios-scpms-starter-mission-06.md

```diff
@@ -54,13 +54,13 @@ To implement the new flow we can attach our current app flow in storyboard to a
 
     !![Split View Controller](fiori-ios-scpms-starter-mission-06-4.gif)
 
-5. Make the Master View Controller the initial view controller for the app by selecting the Master View controller and using the **Attributes Inspector** to check the box for **Is Initial View Controller**.
+5. Make the Main Split View Controller the initial view controller for the app by selecting the View controller and using the **Attributes Inspector** to check the box for **Is Initial View Controller**.
 
     !![Split View Controller](fiori-ios-scpms-starter-mission-06-5.png)
 
     Let's quickly go over what we did here. First we added a new Split View Controller which is a standard Apple UI control. Then we added our app flow to the Split View Controller through a **detail view controller** segue. This allows the Split View Controller to automatically manage your Split View lifecycle. We leave the Navigation Controller in as we want to maintain our regular app flow.
```

Comment:
Please don't say "we", say "you" instead as described in the [style guide](https://jam4.sapjam.com/wiki/show/MozDWaPczznL8NKeGEk0y9?_lightbox=true#mcetoc_1dtqgno1sd).